plugins/node/opentelemetry-instrumentation-pg/README.md

```diff
@@ -0,0 +1,75 @@
+
+This module provides automatic instrumentation for [`pg`](https://github.com/brianc/node-postgres).
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-pg
+```
+
+It will install Instrumentation for PG and PG-POOL
+
+## Usage
+
+To load all of the [default supported plugins](https://github.com/open-telemetry/opentelemetry-js#plugins), use the below approach. Each plugin is only loaded when the module that it patches is loaded; in other words, there is no computational overhead for listing plugins for unused modules.
+
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const provider = new NodeTracerProvider();
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+registerInstrumentations();
+```
+
+If instead you would just want to load a specific instrumentation (**pg** in this case);
+
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const { PostgresInstrumentation } = require('@opentelemetry/instrumentation-pg');
+
+const provider = new NodeTracerProvider();
+const pgInstrumentation = new PostgresInstrumentation();
+pgInstrumentation.setTracerProvider(provider);
+```
+
+PostgresInstrumentation contains both pg and [`pg.Pool`](https://node-postgres.com/api/pool) so it will be instrumented automatically.
+
+See [examples/postgres](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/postgres) for a short example.
+
+### PostgreSQL Instrumentation Options
+
+PostgreSQL instrumentation has few options available to choose from. You can set the following:
+
+| Options | Type | Description |
+| ------- | ---- | ----------- |
+| [`enhancedDatabaseReporting`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-pg/src/pg.ts#L48) | `boolean` | If true, additional information about query parameters and results will be attached (as `attributes`) to spans representing database operations |
+
+## Supported Versions
+
+- [pg](https://npmjs.com/package/pg): `7.x`
+- [pg-pool](https://npmjs.com/package/pg-pool): `2.x` (Installed by `pg`)
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
+- For help or feedback on this project, join us on [gitter][gitter-url]
+
+## License
+
+Apache 2.0 - See [LICENSE][license-url] for more information.
+
+[gitter-image]: https://badges.gitter.im/open-telemetry/opentelemetry-js.svg
+[gitter-url]: https://gitter.im/open-telemetry/opentelemetry-node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+[license-url]: https://github.com/open-telemetry/opentelemetry-js/blob/main/LICENSE
```

Comment:
some urls here should be updated from opentelemetry-js to opentelemetry-js-contrib